doc/source/glossary.rst

```diff
@@ -6,20 +6,27 @@ Glossary
 
 
    (`n`,)
-       A tuple with one element. The trailing comma distinguishes a one-element
-       tuple from a parenthesized ``n``.
+       A parenthesized number followed by a comma denotes a tuple with one
+       element. The trailing comma distinguishes a one-element tuple from a
+       parenthesized ``n``.
 
 
    -1
-       Used as a dimension entry, ``-1`` instructs NumPy to choose the length
-       that will keep the total number of elements the same.
+       - **In a dimension entry**, instructs NumPy to choose the length
+         that will keep the total number of array elements the same.
 
+           >>> np.arange(12).reshape(4, -1).shape
+           (4, 3)
 
-   ``...``
-       An :py:data:`Ellipsis`
+       - **In an index**, any negative value
+         `denotes <https://docs.python.org/dev/faq/programming.html#what-s-a-negative-index>`_
+         indexing from the right.
 
-       **When indexing an array**, shorthand that the missing axes, if they
-       exist, are full slices.
+   . . .
```

Comment:
Any reason for the spaces here? That's likely to frustrate users searching for `...` on this page.
```suggestion
   ...
```

Reply:
Legibility, of course.  I thought it more likely that someone would eyeball the glossary for the ellipsis, especially since it's on top of the page, than make the effort of searching with the browser. 

Sphinx search doesn't find either version. 

Here's how it looks without spaces:

![image](https://user-images.githubusercontent.com/6691888/94345245-8fec0680-fff2-11ea-992a-9b7cd81734f3.png)

I think the spaces help more users than they hurt.